cypress/integration/admin_homepage_top_config.spec.js

```diff
@@ -26,10 +26,10 @@ describe("Update Homepage fields and revert", function() {
 
     cy.get("#content-wrapper > div > div > ul")
       .find(":nth-child(4) > a")
-      .contains("Homepage Top Config")
-      .click();
+      .contains("Homepage Config")
```

Comment:
Why you remove `Top`?

Reply:
Because I renamed that section to just homepage config in my changes for this PR.